community_supplied/Chassis/netconf-chassis.playbook

```diff
@@ -0,0 +1,5 @@
+healthbot {
+    playbook netconf-chassis-playbook {
+        rules chassis.temperature/check-netconf-chassis-temperatures;
```

Comment:
Can we rename this rule to check-netconf-chassis-component-temperature instead?

Reply:
Addressed